### ISSUE TITLE
RegistrationService Artefact Publishing

### ISSUE THREAD

**Issue by Izzzu** [agera-edc/MinimumViableDataspace#165]
Feature #30 
After #164 

## Description
Publish Registration Service artifacts to central artifact repository so that it can be accessed as a dependency by other services based on the dependency management concept described in #164 .

## Acceptance criteria

- [ ] Registration Service artifacts are publicly available and can be used as gradle dependencies
- [ ] MVD repositories use the Registration Service public artifacts instead of publishing them locally in the workflow
- [ ] All documentation/instructions mentioning local publishing artifacts for Registration Service is updated

**Comment by zeier** [comment]
Put on hold, as it is projected to be too much scope for the team until end of August.
#36 and #157 have higher priority.
This item can be taken up again if there is capacity.
